test/hooks/useSelector.spec.js

```diff
@@ -47,6 +47,31 @@ describe('React', () => {
       })
 
       describe('lifeycle interactions', () => {
+        it.only('always uses the latest state', () => {
```

Comment:
`it.only` - only this particular test is executed

Reply:
OMG! Thanks and sorry!